packages/frontend/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM node:12-alpine as build
+
+WORKDIR /
+
+# package.json of root and of needed packages
+COPY package.json .
+COPY yarn.lock .
+COPY babel.config.js .
+COPY packages/frontend/package.json packages/frontend/package.json
+COPY packages/common/package.json packages/common/package.json
+
+# Install at root level
+RUN yarn install --pure-lockfile --non-interactive
+
+# Get src files
+COPY packages/frontend packages/frontend
+COPY packages/common packages/common
+
+ARG NODE_OPTIONS=--max_old_space_size=4096
+RUN yarn workspaces run build
+
+ENV NODE_ENV production
+
+WORKDIR /packages/frontend
+
+FROM nginx:1.16.0-alpine
+COPY --from=build packages/frontend/build /usr/share/nginx/html
+COPY packages/frontend/nginx.conf /etc/nginx/conf.d/default.conf
```

Comment:
Honestly no idea why we need nginx for the frontend, just seems like how infra is set up. We probably need an infra clean up, lot of nginx files floating around when we probably just need one reverse proxy config file and that too only for when we decide to deploy to EC2/Virtual machine. All that being said, I'm going to leave it like so for now since I don't want to touch infra changes in this PR 🤔

Reply:
yea no worries, was just curious. we can investigate later. thanks!